invenio_rdm_migrator/load/postgresql.py

```diff
@@ -81,31 +81,35 @@ def _load(self, table_entries):
                 name = table._table_name
                 cols = ", ".join([f.name for f in fields(table)])
                 fpath = self.tmp_dir / f"{name}.csv"
-                file_size = fpath.stat().st_size  # total file size for progress logging
-
-                print(f"[{_ts()}] COPY FROM {fpath}")  # TODO: logging
-                with contextlib.ExitStack() as stack:
-                    cur = stack.enter_context(conn.cursor())
-                    copy = stack.enter_context(
-                        cur.copy(f"COPY {name} ({cols}) FROM STDIN (FORMAT csv)")
-                    )
-                    fp = stack.enter_context(open(fpath, "r"))
-
-                    block_size = 8192
-
-                    def _data_blocks(block_size=8192):
-                        data = fp.read(block_size)
-                        while data:
-                            yield data
-                            data = fp.read(block_size)
-
-                    for idx, block in enumerate(_data_blocks(block_size)):
-                        if idx % 100:
-                            cur_bytes = idx * block_size
-                            percentage = (cur_bytes / file_size) * 100
-                            progress = f"{cur_bytes}/{file_size} ({percentage:.2f}%)"
-                            print(f"[{_ts()}] {name}: {progress}")
-                        copy.write(block)
+                if fpath.exists():
```

Comment:
ideally a `TableGenerator` would not yield a file name if there is no entries of that table, however that requires more complex logic that maybe can be developed as part of error handling?